sway-core/src/asm_generation/from_ir.rs

```diff
@@ -1380,6 +1400,11 @@ impl<'ir> AsmBuilder<'ir> {
         self.reg_map.insert(*instr_val, base_reg);
     }
 
+    fn compile_int_to_ptr(&mut self, instr_val: &Value, int_to_ptr_val: &Value) {
+        let val_reg = self.value_to_register(int_to_ptr_val);
+        self.reg_map.insert(*instr_val, val_reg);
+    }
```

Comment:
Yep, this will be a conflict with `master` now.

Reply:
All good now :)